scripts/instructions.txt

```diff
@@ -0,0 +1,25 @@
+# Instagram Scraper
```

Comment:
Good idea on documenting the instructions, can we instead turn this into a `README.md` file under the `scripts` directory, and add a section for the `Instagram Scraper` in there?